CODEOWNERS

```diff
@@ -193,18 +181,89 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
+# DNS resolution
+/*/extensions/network/dns_resolver/cares @yanavlasov @mattklein123
+/*/extensions/network/dns_resolver/apple @yanavlasov @mattklein123
+# compression code
+/*/extensions/filters/http/decompressor @rojkov @kbaichoo
+/*/extensions/filters/http/gzip @gsagula @kbaichoo
+/*/extensions/filters/http/common/compressor @gsagula @rojkov @kbaichoo
+/*/extensions/filters/http/compressor @rojkov @kbaichoo
+/*/extensions/compression/brotli @rojkov @kbaichoo
+/*/extensions/compression/common @kbaichoo @rojkov
+/*/extensions/compression/gzip/decompressor @rojkov @kbaichoo
+/*/extensions/compression/gzip/compressor @rojkov @kbaichoo
+/*/extensions/compression/gzip/common @rojkov @kbaichoo
+# cel
+/*/extensions/access_loggers/filters/cel @kyessenov @douglas-reid @adisuissa
+# health cehck
+/*/extensions/filters/http/health_check @mattklein123 @adisuissa
+# lua
+/*/extensions/filters/http/lua @mattklein123 @wbpcode
+/*/extensions/filters/common/lua @mattklein123 @wbpcode
+# rbac
+/*/extensions/filters/network/rbac @yangminzhu @yanavlasov
+/*/extensions/filters/http/rbac @yangminzhu @yanavlasov
+/*/extensions/filters/common/rbac @yangminzhu @yanavlasov
+# tap
+/*/extensions/filters/http/tap @mattklein123 @daixiang0
+/*/extensions/common/tap @mattklein123 @daixiang0
+/*/extensions/transport_sockets/tap @mattklein123 @daixiang0
+# local rate limit
+/*/extensions/filters/network/local_ratelimit @mattklein123 @wbpcode
+# proxy protocol
+/*/extensions/filters/listener/proxy_protocol @ggreenway @soulxu @rgs1
+# access loggers
+/*/extensions/access_loggers/grpc @wbpcode @cpakulski @rgs1 @giantcroc @gyohuangxin
+# stats
+/*/extensions/stat_sinks/statsd @mattklein123 @suniltheta
+/*/extensions/stat_sinks/common @mattklein123 @suniltheta
+/*/extensions/stat_sinks/common/statsd @mattklein123 @suniltheta
+# access loggers
+/*/extensions/access_loggers/file @wbpcode @cpakulski @rgs1 @giantcroc
+# Stateful session
+/*/extensions/http/stateful_session/cookie @wbpcode @cpakulski
+/*/extensions/filters/http/stateful_session @wbpcode @cpakulski
+# tracers
+/*/extensions/tracers/zipkin @wbpcode @Shikugawa @suniltheta
+/*/extensions/tracers/dynamic_ot @wbpcode @Shikugawa
+/*/extensions/tracers/opencensus @wbpcode @Shikugawa
+/*/extensions/tracers/common @wbpcode @Shikugawa
+/*/extensions/tracers/common/ot @wbpcode @Shikugawa
+# ext_authz
+/*/extensions/filters/common/ext_authz @esmet @gsagula @pradeepcrao @ggreenway
+/*/extensions/filters/http/ext_authz @esmet @gsagula @pradeepcrao @ggreenway
+/*/extensions/filters/network/ext_authz @esmet @gsagula @pradeepcrao @ggreenway
+# original dst
+/*/extensions/filters/listener/original_dst @kyessenov @lambdai @lizan
+
+# Maintainerless extensions (need maintainer owner)
+# formatter
+/*/extensions/formatter/req_without_query @UNOWNED @tsaarni @rgs1 @giantcroc
+# IP tagging
+/*/extensions/filters/http/ip_tagging @rgs1 @UNOWNED
+# mongo proxy
+/*/extensions/filters/network/mongo_proxy @mythra @giantcroc @UNOWNED
```

Comment:
I can do stamps on this after @Mythra and @giantcroc since I wrote the original code.

Reply:
SG.  I'll take ip tagging and formatting since @rgs1 has been so helpful to the comunity, and @daixiang0 had offered to help in general so I'll tag him as the second for cors, and we're good!